livekit-android-sdk/src/main/java/io/livekit/android/room/Room.kt

```diff
@@ -938,6 +942,9 @@ constructor(
      */
     override fun onTrackUnpublished(publication: LocalTrackPublication, participant: LocalParticipant) {
         listener?.onTrackUnpublished(publication, participant, this)
+        if (e2eeManager != null) {
```

Comment:
```suggestion
        e2eeManager?.let { e2eeManager ->
```

Removes the need for the `!!` in the following line.